decko/decko.gemspec

```diff
@@ -9,7 +9,7 @@ DeckoGem.gem "decko" do |s, d|
 
   s.files = Dir["{db,lib,public,set}/**/*"]
 
-  s.bindir = "bin"
+  s.bindir = "exe"
```

Comment:
Why?

Reply:
This is the new standard in rails. This is to avoid confusion with bin (old way was binstubs). For a 'bin', something in this dir (exe in the gem, not in files list, this is infra stuff), it creates a bin in gems/bin and in the system (/usr/local/bin, I think) for the entry point that loads bundler and the command from the gem. That stub calls this file. Equivalent is 'exe' for rails now. This stuff does not need the script loading, this is the place for that stuff.